src/plugins/github/graphql.js

```diff
@@ -820,6 +821,6 @@ export function createFragments(): FragmentDefinition[] {
   ];
 }
 
-export function createVariables(repoOwner: string, repoName: string) {
-  return {repoOwner, repoName};
+export function createVariables(repo: Repo): Repo {
```

Comment:
I think that the type that you want here is actually `{+[string]: any}`
or something like that. The fact that the variables are exactly equal to
the fields of `Repo` is a coincidence/implementation detail that may
change over time.